src/components/molecules/Settings/ProjectList/ProjectList/index.tsx

```diff
@@ -19,31 +18,31 @@ export type Props = {
   archived?: boolean;
   projects?: Project[];
   loading?: boolean;
+  withToggle?: boolean;
```

Comment:
I don't really understand the purpose of this prop? You add it here, set the default to true, and then set it to false in the only place this component is being used.
Can you explain why you added this to this component?

Reply:
I'm guessing you were trying to keep the original functionality (aka being able to display both archived and unarchived projects in the same list)? If so, I don't think we need to retain it since the design has changed. Now you are using this as a more simple list, which is fine to simplify this component to meet its case since it isn't an atom and isn't used anywhere else.

Reply:
yes you are right I wanted to keep the code as it is , incase we change back in the future , as I always avoid to remove any block of code.  but if you think so , I will remove it

Reply:
To keep the code clean and easy to understand, I feel like we shouldn't keep unused functionality as much as possible. Also, since @lavalse updated the design to accommodate infinite scrolling, I doubt we will revert back to the old style since we decided infinite scroll is necessary.